var/spack/repos/builtin/packages/kim-api/package.py

```diff
@@ -45,3 +45,11 @@ def patch(self):
         if self.spec.satisfies('%nvhpc'):
             filter_file('-std=gnu', '',
                         'examples/simulators/simulator-model-example/CMakeLists.txt')
+
+    def cmake_args(self):
+        args = [
+            '-DZSH_COMPLETION_COMPLETIONSDIR={0}/zhs_completion.d/'.format(self.prefix),
+            '-DBASH_COMPLETION_COMPLETIONSDIR={0}/bash_completion.d/'.format(
+                self.prefix)
```

Comment:
I think this will be a cleaner solution:

```suggestion
            '-DCMAKE_INSTALL_SYSCONFDIR={0}/etc'.format(self.prefix)
```

BTW, note the typo in the original: `'-DZSH_COMPLETION_COMPLETIONSDIR={0}/zhs_completion.d/'.format(self.prefix),` which should be `'-DZSH_COMPLETION_COMPLETIONSDIR={0}/zsh_completion.d/'.format(self.prefix),` (transposition `zhs` instead of `zsh`)

Reply:
I was not sure if SYSCONFDIR will do the job. But if it does, good for me.